jetson/power_logging/pyproject.toml

```diff
@@ -7,4 +7,6 @@ requires-python = ">=3.11"
 dependencies = [
     "dvc-s3>=3.2.0",
     "pandas>=2.2.3",
+    "pillow>=11.1.0",
```

Comment:
Is this packaged being used anywhere?

Reply:
It an implicit requirement by pytorch to run resnet18